tests/integration/operatorhub/operatorhub_suite_test.go

```diff
@@ -3,11 +3,9 @@ package integration_test
 import (
 	"testing"
 
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"github.com/openshift/odo/tests/helper"
 )
 
 func TestOperatorhub(t *testing.T) {
-	RegisterFailHandler(Fail)
-	RunSpecs(t, "Operatorhub Suite")
+	helper.RunTestSpecs(t, "Operatorhub Suite")
```

Comment:
This one requires `RunSpecs`, but `RunTestSpecs` calls `RunSpecsWithDefaultAndCustomReporters`, will that be alright?

Reply:
yes, I think this was also a mistake where someone forgot to update it to use Reporters.
This is exactly the reason why we should not have duplicated code like this, and instead have one common function that handles common setup.